getusermedia.html

```
@@ -2032,45 +2032,272 @@
+
+        <p>The OverconstrainedError Constructor is the
+        %OverconstrainedError% intrinsic object.  When
+        <code>OverconstrainedError</code> is called as a function
+        rather than as a constructor, it creates and initializes a new
+        OverconstrainedError object. A call of the object as a
+        function is equivalent to calling it as a constructor with the
+        same arguments. Thus the function call
+        <code><b>OverconstrainedError(</b>...<b>)</b></code> is
+        equivalent to the object creation expression <code><b>new
+        OverconstrainedError(</b>...<b>)</b></code> with the same
+        arguments.</p>
+
+        <p>The <code>OverconstrainedError</code> constructor is
+        designed to be subclassable. It may be used as the value of
+        an <code>extends</code> clause of a class definition. Subclass
+        constructors that intend to inherit the specified
+        <code>OverconstrainedError</code> behaviour must include
+        a <code>super</code> call to
+        the <code>OverconstrainedError</code> constructor to create
+        and initialize the subclass instance with an [[\ErrorData]]
+        internal slot.</p>
+
+        <section>
+          <h2>OverconstrainedError ( constraint, message )</h2>
+
+          <p>When the <code>OverconstrainedError</code> function is
+          called with arguments <i>constraint</i> and <i>message</i>
+          the following steps are taken:</p>
+          <ol>
+            <li>If NewTarget is <b>undefined</b>, let <i>newTarget</i>
+            be the active function object, else let <i>newTarget</i>
+            be NewTarget.</li>
+            <li>Let <i>O</i> be
+            OrdinaryCreateFromConstructor(<i>newTarget</i>,
+            <code>"%OverconstrainedErrorPrototype%"</code>, «[[\ErrorData]]»
+            ).</li>
+            <li>ReturnIfAbrupt(<i>O</i>).</li>
+            <li>Assert: Type(<i>constraint</i>) is String.</li>
```

Yeah, just matching message is probably best, although you could throw for `undefined` instead of ignoring it.